site/en/docs/webstore/publish/index.md

```diff
@@ -53,18 +53,57 @@ the ZIP file and manifest depend on the specifics of your item.
   ZIP file in a way that reduces the size of the user download package. For details, see [Reducing
   the size of the user download package][9].
 
-Before you upload your item, you'll be asked to pick a developer account to own your items. Instead
-of your personal account, we suggest using a dedicated account.
-
-## Create a developer account {: #create-a-developer-account }
-
-Before you can publish on the Chrome Web Store, you need to [register as a Chrome Web Store
-developer][10]. We suggest using a new account just for your item instead of your personal account.
-
-If you already host your item in Google Play and you want your Chrome Web Store listing to show an
-"Available for Android" link, your item must have the same name as your Google Play listing, and
-both items must be owned by the same developer account. To transfer your Chrome Web Store item to a
-different developer, submit this [form][11].
+## Create and setup a developer account {: #create-setup-a-developer-account }
+
+Before you can publish your item on the Chrome Web Store, you'll be asked to pick a developer account to own your items and setup your developer account. 
```

Comment:
A few issues here:

- "you'll be asked" uses both [future tense](https://developers.google.com/style/tense) (will) and [passive voice](https://developers.google.com/style/voice) (be asked), both of which we avoid where possible.
-  "a developer account to own your items and setup your..." seemed confusing.

My suggestion:

"...Store, you need to pick a developer account to own your items, then set up your account in the Chrome Web Store.

Reply:
Got it!